Extensions/xkit_patches.js

```diff
@@ -2221,6 +2221,23 @@ XKit.tools.getParameterByName = function(name){
 				XKit.tools.add_function(function(){
 					Tumblr.Events.trigger("peepr-open-request", add_tag);
 				}, true, payload);
+			},
+
+			/**
+			 * Determines whether a user is following the given blog.
+			 * The logged-in user must be a member of the given blog to determine this.
```

Comment:
What's the behavior if the logged-in user ISN'T a member of the blog? Do we need an error handler here?